README.md

```diff
@@ -1,59 +1,228 @@
--------
-### ⚠️  PLEASE READ THE [INSTRUCTIONS](/INSTRUCTIONS.md) FOR GUIDELINES ON HOW TO START YOUR PACKAGE.
-> Don't forget to remove this warning while updating this README.
--------
-
-# {package-name}
+# react-contentful-image
 
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-contentful-image
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-contentful-image.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-contentful-image.svg
+[build-status-url]:https://github.com/moxystudio/react-contentful-image/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-contentful-image/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-contentful-image
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-contentful-image/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-contentful-image
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-contentful-image.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-contentful-image?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-contentful-image.svg
 
-{package-description}
+A react image rendering using Contentful Images API.
```

Comment:
This sounds a bit off 🤔

Suggestions:

- A react image renderer that uses the Contentful Images API.
- A react component that renders an image using the Contentful Images API.


Feel free to disregard this comment if you don't agree.

Reply:
It's really weird. It's probably a mix between two sentences I wrote.